area.java

```diff
@@ -1,59 +1,71 @@
 import java.util.*;
+class Area {
 
+  public static void main(String[] args) {
+    Scanner area = new Scanner (System.in);
 
-class Main {
+    //user input for the sides of square
+    System.out.println("Write the side of the square");
+		int side = area.nextInt();
+    int squ = square(side);
 
-  public static void main(String[] args) {
+    System.out.println("The area of square is " + squ);
 
-    shapes();
 
-  }
+    ///user input for the base and height of triangle
+    System.out.println("What's the base of the triangle");
+		int base = area.nextInt();
+    System.out.println("What's the height of the triangle");
+		int height = area.nextInt();
+    int tri = triangle(base, height);
 
+    System.out.println("The area of triangle is " + tri);
 
 
-    public static void shapes(){
+    ///user input for the width and length of rectangle
+    System.out.println("What's the width of the triangle");
+		int width = area.nextInt();
+		System.out.println("What's the Length of the triangle");
+		int Length = area.nextInt();
+    int rec = rectangle(width, Length);
 
+    System.out.println("The area of rectangle is " + rec);
 
 
-    Scanner area = new Scanner (System.in);
-    System.out.println("Choose type of conversion \n 1.Square  \n 2.Triangle \n 3.Rectangle \n 4.Circle");
-    int b = area.nextInt();
+    ///user input for the radius of circle
+    System.out.println("What's the radius of the circle");
+		double radius = area.nextDouble();
+    double cir = circle(radius);
 
+    System.out.println("The area of circle is " + cir);
 
-    if(b == 1) {
 
-		System.out.println("Write the side of the square");
-		int side = area.nextInt();
-		int area1 = side * side;
-		System.out.println(area1);
-	}
+  }
 
 
-    else if(b == 2) {
-		System.out.println("What's the base of the triangle");
-		int base = area.nextInt();
-		System.out.println("What's the height of the triangle");
-		int height = area.nextInt();
-		int area2 = ((base * height)/2);
-		System.out.println(area2);
-	}
+    public static int square(int side){
+		  int area1 = side * side;
```

Comment:
To nitpick for a second, let's think about these variable names - remember that the variable only stays defined within the curly brackets, so do you need to call them `area1`, `area2`, etc.? Or can you call them something different?